src/main/webapp/pages/createUpdateIdea.html

```diff
@@ -1,10 +1,27 @@
 <form id="mainDiv" name="ideaForm" novalidate>
     <div id="shadowBox">
         <div id="topDiv">
-            <p id="p" translate="ADD_IDEA"></p>
+            <p id="p" translate="{{windowTitle}}"></p>
         </div>
         <div id="dataDivInsert">
             <div id="lineDivsLeftInsert">
+
+                <!-- State -->
+                <div id="divData" class="form-group" ng-class="{ 'has-error' : ideaForm.edtState.$invalid && (submitted || ideaForm.edtState.$dirty)}" ng-show="{{isUpdating}}">
+                    <div id="tdCaption" class="requiredField">
+                        <label>{{ 'Idea_State' | translate }}</label>
+                    </div>
+
+                    <div id="tdDataInsert" class="controls form-inline">
+                        <!--select class="form-control" ng-model="data.state"-->
+                        <select class="form-control" ng-model="data.state" ng-options="states.indexOf(state) as state for state in states"></select>
+
+                        <span class="text-danger ng-scope ng-active" ng-show="submitted || ideaForm.edtState.$dirty" ng-messages="ideaForm.edtState.$error">
+                            <span ng-message="required"><span id="ideaRequiredMessage" translate="STATE_REQUIRED"/></span>
+                        </span>
+                    </div>
+                </div>
+
                 <!-- idea -->
```

Comment:
move code related to display states into directive (for training purposes) (see show-idea, 'showIdea' as example)